content/post/Android_Root101.en.md

```diff
@@ -49,16 +50,18 @@ Place this file inside the folder created after extraction.
 > `cd..` is used to go down one directory
 
 
-Finally pick your phone up and go to the fastboot mode, two methods have been listed below use any:
+Finally, enter fastboot mode on your phone. How-to methods have been listed below(Use any one of the following):
 
-    1. Reboot to Bootloader ->(Hold Power + Volume down)
+1. Reboot to Bootloader ->(Hold Power + Volume down)    {*_For Xiaomi Devices_*}
 
-    2. Connect your phone to the PC after enabling the ADB debugging mode in the developer settings
+2. Connect your phone to the PC after enabling the ADB debugging mode in the developer settings
 
 > type `adb devices` if your device shows up then,
```

Comment:
type!?
```suggestion
> Use `adb devices` when your device shows up 
```
maybe add troubleshooting steps if it doesn't show up

Reply:
Also, where do you type?
in the phone? is there a command line?
in the pc? in cmd?
better to mention that